code/datums/uplink/ammunition.dm

```diff
@@ -73,4 +73,76 @@
 	name = "K2557 Magazine"
 	item_cost = 2
 	path = /obj/item/ammo_magazine/super_heavy
-	desc = "A spare magazine, for the super heavy K2557 pistol."
\ No newline at end of file
+	desc = "A spare magazine, for the super heavy K2557 pistol."
+
+/datum/uplink_item/item/ammo/shotgun_slug
+	name = "Shotgun Slug"
+	item_cost = 2
```

Comment:
a single slug for 2 TC?

Reply:
Slugs are pretty strong. Maybe I'll lower it to 1.

Reply:
A single slug though? A hacked protolathe can produce them for a few sheets or less of metal. If it is a package, 1 TC would be good. You can only carry so many slugs.